packages/pwa-kit-runtime/src/utils/loggerFactory.js

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
What is the reason for using lower camel case for this file (`loggerFactory.js`) vs using kabob case to be consistent with the rest of the package?